src/util/shmem/fd_shmem.h

```diff
@@ -503,8 +503,11 @@ fd_shmem_page_sz_to_cstr( ulong page_sz );
 /* These functions are for fd_shmem internal use only. */
 
 void
-fd_shmem_private_boot( int *    pargc,
-                       char *** pargv );
+fd_shmem_boot( int *    pargc,
```

Comment:
We don't really change util code like this, it's an API for other consumers outside Firedancer. You should use the API that exists already, aka. passing in a --shmem-path.